.gitmodules

```diff
@@ -7,3 +7,6 @@
 [submodule "sources-api-go"]
 	path = sources-api-go
 	url = https://github.com/RedHatInsights/sources-api-go
+[submodule "notifications-backend"]
+	path = notifications-backend
+	url = https://github.com/amirfefer/notifications-backend.git
```

Comment:
Can we attempt to solve the problem with Maven rather than making a fork? This will make updating the projects much more inconvenient and also if we forget about it we will end up using pretty old version of notifications. There must be way to workardound it somehow.

In the worst case, create a script that will modify the files via `sed` or `aws` and put it into instructions to run it.

Reply:
It is possible to skip the commit id with `-Dmaven.gitcommitid.skip=true`, however, this also requires a change in the `Dockerfile`. I asked on Slack if there is other way.